test/fw/ptl/utils/pbs_anonutils.py

```diff
@@ -734,19 +743,176 @@ def __delete_kv(self, line, key, value):
 
         return line
 
-    def __anon_hosts(self, line):
+    def __add_alias_attr(self, key, alias_key):
         """
-        (WORK IN PROGRESS)
-        Anonymize all hostnames present in the line given
+        Some attributes have aliases. Added alias for a given attribute to the
+        global maps
+
+        :param key: the original attribute
+        :type key: str
+        :param alias_key: the alias
+        :type alias_key: str
         """
+        if key is None or alias_key is None:
+            return
```

Comment:
You should either always return values or never return values.  It's poor python style to sometimes return values and other times not.